src/components/form/wizard/footer.tsx

```diff
@@ -18,22 +18,37 @@ const StyledFooter = styled.div`
     width: 100%;
 `
 
+const enum ButtonType {
+    Previous = 'PREVIOUS',
+    Next = 'NEXT',
+}
+
 const Footer = ({ disabled }: { disabled: boolean }) => {
-    const { step, setStep, max_step } = React.useContext(StepContext)
+    const { step, setStep, max_step, setEnableNext } = React.useContext(StepContext)
+
+    const buttonHandler = (button_type: ButtonType): void => {
```

Comment:
can you please use `useCallback` here? 
since you are getting some values from context, it may re-generate the function again on every render.